doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -325,7 +351,7 @@ NOTE: `curl` is a Unix command allowing for interacting with REST APIs such as t
 | enable\_encoding | no | false | <i>true</i> or <i>false</i>, <i>true</i> applies the new encoding, <i>false</i> applies the old encoding. ||
 | enable\_grouping | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_grouping_interceptor.md) for more details. ||
 | enable\_name\_mappings | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_name_mappings_interceptor.md) for more details. ||
-| data\_model | no | dm-by-entity |  Always <i>dm-by-entity</i>, even if not configured. ||
+| data\_model | no | dm-by-entity | dm-by-entity-id | <i>dm-by-entity</i>, even if not configured. ||
```

Comment:
| data\_model | no | dm-by-entity | <i>dm-by-entity-id</i>, <i>dm-by-entity</i> ||
```suggestion
| data\_model | no | dm-by-entity | dm-by-entity-id | <i>dm-by-entity</i>, even if not configured. ||
```

Reply:
Solved in aad6fe1d70613f27b5bae4ab36969c8fa512c61c